auction-server/migrations/20240910082426_bid_metadata.up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE chain_type AS ENUM ('evm', 'svm');
+
+ALTER TABLE bid
+ADD COLUMN metadata JSONB,
```

Comment:
I agree that adding a transaction column or creating a separate table would make life easier for analysis, but we may lose some functionality like filtering bids of a specific profile based on a time range (Or it will be harder to handle because we need to merge the result of two different tables). Additionally we need to rewrite db queries for svm stuff again and we can't reuse the prv codes.

Additionally I think we can fetch the jsonb fields with sql queries. Could you please explain the analytics use case that you are concerned about?